### ISSUE TITLE
SidekiqUniqueJobs::UniqueArgs#create_digest is getting called twice

### ISSUE THREAD

**Issue by tacticiankerala** [mhenrixon/sidekiq-unique-jobs#391]
**Describe the bug**

I have patched `SidekiqUniqueJobs::UniqueArgs#create_digest` in my project to support different tenants that I have. But, noticed that this method is getting called twice and the interesting bit is it is getting called the second time even when `unique_digest` is there in the job.

ps: Sidekiq worker we have does not take any args hence was unable to use `unique_args` option. Also not wanted to patch it & modify all workers to calculate unique args based on tenant.

**Expected behavior**

I think this method should not get called when the job already has  `unique_digest` value

**Current behavior**

It is getting called more than once even when the job already contains a `unique_digest` and the second time `Apartment::Tenant.current` is incorrect.

**Worker class**

```ruby
module SidekiqUniqueJobs
  class UniqueArgs
    def create_digest_with_apartment
      "#{create_digest_without_apartment}_#{Apartment::Tenant.current}"
    end

    alias_method :create_digest_without_apartment, :create_digest
    alias_method :create_digest, :create_digest_with_apartment
  end
end


class MyWorker
  include Sidekiq::Worker
  sidekiq_options unique: :until_executing, log_duplicate_payload: true

  def perform
   # do something in Redis or Elasticsearch
  end
end

```

**Additional context**
Not sure if this is a use case that would happen very often. So, I don't think this is a super critical issue. Though, I would like to know why this method is getting called the second time though

**Comment by mhenrixon** [comment]
I would really love to get to the bottom of this one. I am just right in the middle of finalizing the next version. 

Since it seems like sort of an edge case I’ll leave it open until then. I’d be interested in digging into how apartment works at some point. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by mhenrixon** [comment]
You are right. I know what to do about this now. Thank you for reporting

**Comment by tacticiankerala** [comment]
This is not a blocker by any means. But, a nice to have to be fixed. Let me know if you need anything.

**Comment by mhenrixon** [comment]
I believe this was fixed for v7.0.1; see #571 for how to upgrade.
